rcl_yaml_param_parser/test/test_parse.cpp

```diff
@@ -436,16 +436,6 @@ TEST(TestParse, parse_key_bad_args)
   EXPECT_TRUE(rcutils_error_is_set());
   rcutils_reset_error();
 
-  // map_level is not a valid value
-  map_level = 42;
```

Comment:
Now this can be detected by the compiler during build because it checks the enum data, that is why we do not even need to have this test after all.